code/modules/events/solarflare.dm

```diff
@@ -0,0 +1,10 @@
+/datum/event/solar_flare
+	startWhen = 2
+	endWhen = 3
+	announceWhen = 1
```

Comment:
Are there no defines for these?

Reply:
They're a counter for the number of ticks. Not mode switches.